test-case/verify-kernel-module-load-probe.sh

```diff
@@ -20,7 +20,9 @@ DMESG_LOG_START_LINE=$(sof-get-kernel-line.sh|tail -n 1 |awk '{print $1;}')
 
 dlogi "Checking if sof relative modules loaded"
 dlogc "lsmod | grep \"sof\""
-lsmod | grep "sof"
-[[ $? -ne 0 ]] && dloge "No available sof module found! Dumping lsmod:" && lsmod && exit 1
+lsmod | grep "sof" || {
+    lsmod
+    die "No available sof module found!"
+}
```

Comment:
already catch 'sof' need `lsmod` again? or need some output

Reply:
looks `lsmod `is added for more debugging, not very useful but it is paired with original file.

Reply:
This PR is a very large search/replace so let's make sure there is no functional changes (hidden) in it.